core/math/geometry_3d.cpp

```diff
@@ -629,7 +629,10 @@ Geometry3D::MeshData Geometry3D::build_convex_mesh(const Vector<Plane> &p_planes
 			}
 
 			for (uint32_t k = 0; k < vertices.size(); k++) {
-				int k_n = (k + 1) % vertices.size();
+				uint32_t k_n = k + 1;
```

Comment:
Why uint32_t instead of int?

Reply:
To avoid a warning about comparing signed, unsigned int. And also `k` is a `uint32_t`.